justfile

```diff
@@ -0,0 +1,284 @@
+# Configuration
+# set shell := ["bash", "-uc"]
```

Comment:
I like the idea of using the same shell consistently. Are there particular lines that you think would benefit from zsh features?

```suggestion
set shell := ["bash", "-uc"]
```